_config.yml

```
@@ -0,0 +1,9 @@
+theme: jekyll-theme-primer
+encding: utf-8
+plugins:
+  - 'jekyll-twitter-plugin'
+repository: mixi-git-challenge/publications
+destination: docs
+baseurl: publications
+exclude:
+  - dev
```

OGP周りの追記が必要そう
```suggestion
  - dev
title: git challenge publication
twitter:
  username: hr_mixi
  card: summary_large_image
facebook:
  app_id: XXX
```

必要そうなタグ今すぐには洗い出せないからちょっとあとでみる
https://jekyll.github.io/jekyll-seo-tag/usage/

---

とりあえず、jekyll-seo-tag の設定を enable にして、title と twitter を足した